src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
better not to discard all the margins and paddings 🙃

Reply:
you are right 😎, but on larger pages this will be useless, because you btw will set margins for elements 😉